subsys/bluetooth/host/hci_core.c

```diff
@@ -61,10 +61,14 @@
 #define HCI_CMD_TIMEOUT      K_SECONDS(10)
 
 /* Stacks for the threads */
-#if !defined(CONFIG_BT_RECV_IS_RX_THREAD)
-static struct k_thread rx_thread_data;
+#if !defined(CONFIG_BT_RECV_BLOCKING)
```

Comment:
We could consider doing 
```suggestion
#if defined(CONFIG_BT_RECV_WORKQ_SYS) || defined(CONFIG_BT_RECV_WORKQ_BT)
```
here as well, but I'll leave that up to you. It is OK as is.